linode/image/resource_test.go

```diff
@@ -208,6 +215,47 @@ func TestAccImage_uploadFile(t *testing.T) {
 	})
 }
 
+//
+//func TestAccImage_replicate(t *testing.T) {
```

Comment:
Uncomment this test case once replication is functional.

Reply:
I would recommend using `t.Skip("some reasons")` rather than comment it out.